crates/alerter/Cargo.toml

```diff
@@ -24,3 +24,6 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
 url = { workspace = true }
 warp = { workspace = true }
+
+[lib]
+doctest = false
```

Comment:
Is it required here to address the initial issue, or can this be separated from the PR?

Reply:
It can be removed. Unrelated to this PR entirely. The addition was to prevent running of additional tests where there are none, which also reduces the volumes of output when running the tests.

If you'd like I can remove it.

Reply:
+1 for removing it (as unrelated)

Reply:
Yes, please. Thanks, it would be nice to have it as a separate PR.